.circleci/config.yml

```diff
@@ -70,9 +82,46 @@ jobs:
       - run: make build
       - check-uncommitted
       - *save_cache
+      - persist_to_workspace:
+          root: *workspace_root
+          paths:
+            - .
```

Comment:
Is there a need to copy everything over? Or is `dist/` enough?

Reply:
Probably just `dist` -- any concerns with copying everything over for simplicity?

Reply:
I'm 0/5 here. A couple of small concerns are:
- Performance
- Conflicts with other files in the future
- Copying just `dist` is "the right thing" :tm:

Reply:
I agree with the gist of all of that -- I'm inclined to be lazy here and "get this working", and then consider tweaking later if we find the performance unacceptable or causing conflicts. There was a bunch of churn even getting the files to share across the workspaces, that I'm hesitant to mix it up again. 2/5